test/src/unit/repo_test_utils.h

```diff
@@ -197,3 +199,133 @@ static std::string getRandomString(const uint32_t &iLen)
 
 	return sStr;
 }
+
+namespace repo
+{
+namespace test
+{
+	class TestLogging
+	{
+	public:
+
+		static void printTestTitleString(std::string title, std::string description)
```

Comment:
@carmenfan, not sure if this is the best place to put this function, would it would be placed alongside the repo errors h file. Or did I just bungle something up and we already had something like this?

Reply:
I don't think we need it really, That's what the 2nd parameter of TEST is for.

If you need it to be more precise then just split them into multiple test cases, with more descriptive test names.

Logging should be kept to minimum and precise ideally. Otherwise it clutters the field and it begins to get extremely hard to read.

Reply:
If you really need an explanation of what the test is doing, put it as a code comment.

When you're running the test you don't really care unless it stops working, if it stops working you'd have to refer to the code anyway.

Reply:
You know what, that makes total sense, I always used to try and make the test outputs clear as to what they were supposed to be doing, but ultimately no one cares until something goes wrong. I'll refactor and put any useful statements as code comments.